tutorials/scripting/gdscript/static_typing.rst

```diff
@@ -396,6 +396,53 @@ if you prefer a more readable and reliable, but more verbose syntax.
 ``UNSAFE_*`` warnings make unsafe operations more noticeable, than unsafe lines.
 Currently, ``UNSAFE_*`` warnings do not cover all cases that unsafe lines cover.
 
+Common unsafe operations and their safe counterparts
+----------------------------------------------------
+
+Warning: ``UNSAFE_CAST``
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the example below, we would like an object entering our collision area to take
+damage if it has a health variable that contains a ``Health`` object. The physics
```

Comment:
I think most often the `health` variable is of type `int` or `float`. Perhaps this example is not very successful in a practical sense. I often cite the `_input()` method and the `InputEvent` and `InputEventMouseMotion` classes as an example of appropriate use of the `as` operator.

Reply:
Good point; came up with another scenario.